src/Nancy.Hosting.Self/project.json

```diff
@@ -0,0 +1,46 @@
+{
+  "version": "1.0.0-*",
+  "description": "",
+  "authors": [ "" ],
+  "tags": [ "" ],
+  "projectUrl": "",
+  "licenseUrl": "",
+
+  "dependencies": {
+    "Nancy": "1.0.0-*",
+    "Microsoft.Framework.Runtime.Abstractions": "1.0.0-beta5-*"
+  },
+  "compileFiles": [ "../SharedAssemblyInfo.cs" ],
+  "frameworks": {
+    "dnx451": {
+      "frameworkAssemblies": {
+        "System.Diagnostics.Debug": ""
+      }
+    },
+    "dnxcore50": {

```

Comment:
Maybe we should kill these until we have a better story on .NET core? Seems like this PR is biting off a bit more than it can chew.